week4_approx/atari_util.py

```diff
@@ -1,59 +1,62 @@
 """Auxilary files for those who wanted to solve breakout with CEM or policy gradient"""
 import numpy as np
-import gym
-from scipy.misc import imresize
 from gym.core import Wrapper
 from gym.spaces.box import Box
+from skimage.transform import resize
+
 
 class PreprocessAtari(Wrapper):
-    def __init__(self, env, height=42, width=42, color=False, crop=lambda img: img, 
-                 n_frames=4, dim_order='theano', reward_scale=1,):
+    def __init__(self, env, height=42, width=42, color=False,
+                 crop=lambda img: img, n_frames=4, dim_order='theano'):
```

Comment:
This _may_ be a problem. I will need to check whether `reward_scale` is actually used wherever this script is imported from.

Reply:
`week4_approx/dqn_atari.ipynb` doesn't even import `atari_util`, no idea why the file is even there.